pyartifactory/models/repository.py

```diff
@@ -40,6 +41,7 @@ class PackageTypeEnum(str, Enum):
     rpm = "rpm"
     sbt = "sbt"
     terraform = "terraform"
+    terraformbackend = "terraformbackend"

```

Comment:
I cannot see terraformbackend form the [Public API](https://jfrog.com/help/r/jfrog-rest-apis/repository-configuration-json). Can you please point me to where it is documented ?